reports/spec3-alloy.tex

```diff
@@ -14,6 +14,7 @@ \section{\SpecThreeB{} in Alloy}\label{sec:alloy}
       restrict the number of checkpoints and votes to 5 and 12, respectively.
       Although we introduced similar restrictions with Apalache, Alloy has even
       finer level of parameter tuning.
+      \rs{Is there some inherent limitation of Apalache that we should discuss?}
```

Comment:
No inherent limitation per se, there is just no finer-grained control that Apalache implements or exposes right now. We actually mention it as a possible extension in §1.3.

Reply:
Yes, we touched upon that in para 1.3. Beyond that, it is hard to explain the difference without explaining the technique implemented by Alloy.